package.json

```diff
@@ -42,7 +42,7 @@
     "xlsx": "^0.18.5"
   },
   "devDependencies": {
-    "@nestjs/cli": "^10.0.0",
```

Comment:
Please don't update this library version because is not tested yet in the dev app

Reply:
Done